package.json

```diff
@@ -32,7 +32,7 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-react": "^7.16.0",
     "eslint-plugin-standard": "^4.0.1",
-    "extract-text-webpack-plugin": "^4.0.0-beta.0",
+    "mini-css-extract-plugin": "^0.9.0",
```

Comment:
:+1: We already did that in https://github.com/cockpit-project/cockpit/commit/dabebcab33d7 

Let's please do that in starter-kit, too.

Reply:
Ah nice, was not aware that this is already in Cockpit. I'll do that in starter kit and send the logging suppression into cockpit.